src/ASM/ASMbase.C

```diff
@@ -21,6 +21,10 @@
 #include "Utilities.h"
 #include <algorithm>
 
+#ifdef USE_OPENMP
+#include <omp.h>
+#endif
+
```

Comment:
vel, den sjekken må i så fall sitte i SparseMatrix, og den må sjekke typen for en dof for om den skal gjøre det eller ikke. høres ikke ut som en veldig fornuftig løsning.